libraries/AP_VisualOdom/AP_VisualOdom_IntelT265.cpp

```diff
@@ -0,0 +1,199 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "AP_VisualOdom_IntelT265.h"
+#include <AP_HAL/AP_HAL.h>
+#include <AP_AHRS/AP_AHRS.h>
+#include <AP_Logger/AP_Logger.h>
+
+extern const AP_HAL::HAL& hal;
+
+// constructor
+AP_VisualOdom_IntelT265::AP_VisualOdom_IntelT265(AP_VisualOdom &frontend) :
+    AP_VisualOdom_Backend(frontend)
+{
+}
```

Comment:
```suggestion
using AP_VisualOdom_Backend::AP_VisualOdom_Backend;
```

(actually, that bit goes in the header, just remove this definition)

Reply:
OK, changed although this is really just following the existing pattern that I think you wrote in the GCS_MAVLink library. https://github.com/ArduPilot/ardupilot/blame/master/libraries/GCS_MAVLink/GCS_Common.cpp#L2984

Reply:
ok, changed to use using